src/EditorFeatures/Core.Wpf/InlineRename/UI/SmartRename/SmartRenameControl.xaml.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Input;
+
+namespace Microsoft.CodeAnalysis.InlineRename.UI.SmartRename
+{
+    /// <summary>
+    /// Interaction logic for SmartRenameControl.xaml
+    /// </summary>
+    internal partial class SmartRenameControl : UserControl
+    {
+        private readonly SmartRenameViewModel _smartRenameViewModel;
+        internal SmartRenameControl(SmartRenameViewModel viewModel)
+        {
+            this.DataContext = _smartRenameViewModel = viewModel;
+            InitializeComponent();
+        }
+
+        private void Suggestion_MouseDown(object sender, MouseButtonEventArgs e)
```

Comment:
In LSP UI, single click updates `SelectedSuggestedName` and double click commits the session. This is a bit different. We can discuss it with PMs

Reply:
I changed it to update the content with one click. 
Personally I don't like `commit` when double click, because sometimes I just misclick one more time in the UI and the rename get committed. But yeah we can ask PM's advice here